src/collectd_dogstatsd.py

```diff
@@ -181,11 +178,15 @@ def send_points(self, metrics):
 
             val.type_instance = metric['metric']
             val.plugin_instance = combine_dims(dims_from_tags(metric['tags']))
+            parsed_time = int(metric['points'][0][0])
+            if parsed_time > 0:
+                val.time = parsed_time
```

Comment:
Is the collectd time format the same as what dogstatsd points use?

Reply:
I assume so, in the SignalFx point sender we were converting to ms.  collectd expects seconds since epoch.